pkg/api/v1beta3/types.go

```diff
@@ -971,6 +971,16 @@ type NodeSystemInfo struct {
 	SystemUUID string `json:"systemUUID"`
 	// BootID is the boot-id reported by the node
 	BootID string `json:"bootID" description:"boot id is the boot-id reported by the node"`
+	// Kernel version reported by the node
+	KernelVersion string `json:"kernelVersion" description:"Kernel version reported by the node"`
+	// OS image used reported by the node
+	OsImage string `json:"osImage" description:"OS image used reported by the node"`
```

Comment:
I was going to ask this as well. My guess was something like `uname -o`, but I think @dchen1107 may have had in mind the actual userland image version. If the latter, I'm not sure it has a meaning for all cases, and we may want to split this up into `uname -o` and a "version".